### ISSUE TITLE
I want accessibility tools front &amp; center

### ISSUE THREAD

**Issue by aarongustafson** [WebWeWant/webwewant.fyi#194]
Accessibility tools should be center stage, not buried. Most Dev Tools are good and getting better at evaluating and presenting programmatically detectable issues. But that is a process that is manually initiated in a near-hidden tab. The idea of “inspect element” should not be limited to the declared style and compiled output. It should first indicate an accessibility status.

&gt; Dear human inspecting this element, before you try to unveil what CSS features and properties have been applied, please note this fails {x.x.x} criterion of WCAG, and requires further inspection of {consideration}.

Accessibility needs more visibility (pun intended). Dev Tools can help.

<hr>

https://webwewant.fyi/wants/12/
